### ISSUE TITLE
FileNotFound followed by CircularDependency

### ISSUE THREAD

**Issue by beverlycodes** [sstephenson/sprockets#133]
Using **v2.0.0.beta.11** with **Rails 3.0**

I have my javascripts in the following folder structure:
- app/
  - javascripts/
    - application.js
  - views/
    - staff_members/
      - show.js

In setting up the Rack app, I've used the following:

``` javascript
  def initialize
    super Rails.root.join('app')
    append_path('javascripts')

    if Rails.env.production?
      self.js_compressor = YUI::JavaScriptCompressor.new :munge => true, :optimize => true
      self.css_compressor = YUI::CssCompressor.new
    end
  end
```

When I attempt to `//= require <views/staff_members/show>` from application.js, I end up with `Sprockets::FileNotFound: couldn't find file '<views/staff_members/show>'\n  (in /Users/crimsonfox/RailsApps/gatescan/app/javascripts/application.js:1)`

All subsequent attempts to load application.js result in `Sprockets::CircularDependencyError: /Users/crimsonfox/RailsApps/gatescan/app/javascripts/application.js has already been required` until I restart my Rails server process.

Rails caching is current turned off.

I've noticed that I can call `resolve` within `initialize`, passing it 'views/staff_members/show' and it will return the correct path to the file.  I'm not sure why the file can't be found via `require <views/staff_members/show>`.

To summarize, on a fresh restart I get FileNotFound and refreshes result in CircularDependencyError until a restart.  However, if I use the double-quote method of `require`, I get the expected result.  I'm fine with using the work-around.  I'd just like to know if there's a bug in the other method, or if I've missed something critical in my understanding of how it works.
